acp/acp_source_hub.go

```diff
@@ -8,6 +8,8 @@
 // by the Apache License, Version 2.0, included in the file
 // licenses/APL.txt.
 
+//go:build !js
```

Comment:
suggestion: Although obvious why now, might be a headscratcher later, please document the reasoning.

Reply:
Added a comment here.